test/cli/tsconfig.spec.ts

```diff
@@ -20,7 +20,15 @@ afterEach(async () => {
 const locate = (project: string | undefined, fileNames: string[] = []) =>
     locateConfigFile({ errors: [], fileNames, options: { project } });
 
-const normalize = (name: string) => normalizeSlashes(path.resolve(temp, name));
+const normalize = (name: string) => {
+    const result = normalizeSlashes(path.resolve(temp, name));
+
+    if (process.platform === "darwin") {
```

Comment:
Please revert to what it was before, that was much cleaner, but put the realPathSync call inside normalizeSlashes instead of ourside to prevent windows breaking

Reply:
Sure thing. sorry about that.